pkg/octavia/pem_decrypt_fips.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2012 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
Is the license file for the operator the same as the license in the GOlang?
If not, you probably need to paste the golang license here.

Reply:
No longer relevant. I am no longer copying code from the Golang stdlib.